lib/NormalModuleFactory.js

```diff
@@ -299,6 +299,7 @@ class NormalModuleFactory extends Tapable {
 	resolveRequestArray(contextInfo, context, array, resolver, callback) {
 		if(array.length === 0) return callback(null, []);
 		asyncLib.map(array, (item, callback) => {
+			const originLoaderName = item.loader;
```

Comment:
if this is called `origin` further on why not keep in sync and go for `const origin = item.loader;`
and further on 
```
Object.assign({ origin }, item, identToLoaderRequest(result), optionsOnly));
```
Its not that i mind too much but looks much cleaner

Reply:
Right, that's neat!